modules/Kconfig.mcux

```diff
@@ -388,6 +388,12 @@ config NXP_MONOLITHIC_BT
 	   If enabled, the BT firmware used by the device will be linked with the
 	   application directly.
 
+config NXP_MONOLITHIC_IEEE802154
+	bool "IEEE802154 firmware monolithic build"
```

Comment:
Zephyr supports only monolithic BT & WiFi.
Should it be set to y, by default?

Reply:
Sorry, this is to support MFG builds, which are not needed upstream, so I will close this PR.